edx_notifications/channels/urban_airship.py

```diff
@@ -77,19 +77,21 @@ def create_payload(msg, user_id):
 
         return obj
 
-    def call_ua_push_api(self, payload):
+    def call_ua_push_api(self, payload, auth_dict):
```

Comment:
`auth_dict` should be renamed to `api_keys` should be a pass in via `channel_context` parameter because we should not change the signature of `bulk_publish_notification_to_users ` method.